frontend/src/components/ui/DifficultyButtonGroup.js

```diff
@@ -0,0 +1,21 @@
+import { ButtonGroup, Container } from "@mui/material";
```

Comment:
Just a nitpick, for consistency all of our material ui component import are in the form of:
`import <Component Name> from '@mui/material/<Component Name>'`.

So, if it's not troublesome, can you help change to:
`import Box from "@mui/material/Box";`

Thanks alot!

Reply:
Sure thing, thanks for pointing out!